migrations/tests/extensions/10-timescaledb.sql

```diff
@@ -1,7 +1,7 @@
 begin;
 do $_$
 begin
-  if not exists (select 1 from pg_extension where extname = 'orioledb') then
+  if current_setting('server_version_num')::integer >= 150000 and current_setting('server_version_num')::integer < 160000 then
```

Comment:
I had to exclude orioledb before since it gave errors when running with timescale. This conditional implies that orioledb will now work normally starting from pg 15?

Reply:
This is for creating extension timescaledb and not oriole fwiw

Reply:
This branch contains both pg 17 and orioledb-17 and so it's better to just limit this logic to pg 15, as that is the only major version that supports these extensions that this file is trying to create extension on (same with plv8)

Reply:
Ok cool, got it. Thanks for elaborating.